src/Features/Core/Portable/PdbSourceDocument/PdbSourceDocumentMetadataAsSourceFileProvider.cs

```diff
@@ -257,7 +257,7 @@ internal sealed class PdbSourceDocumentMetadataAsSourceFileProvider : IMetadataA
             // to the document passed in, which we just use the first document for.
             // TODO: Support results from multiple source files: https://github.com/dotnet/roslyn/issues/55834
             var firstDocumentFilePath = sourceFileInfos[0]!.FilePath;
-            var firstDocument = navigateProject.Documents.FirstOrDefault(d => d.FilePath?.Equals(firstDocumentFilePath, StringComparison.OrdinalIgnoreCase) ?? false);
+            var firstDocument = navigateProject.Documents.First(d => d.FilePath?.Equals(firstDocumentFilePath, StringComparison.OrdinalIgnoreCase) ?? false);
```

Comment:
@davidwengier Does this look right to you>

Reply:
i'd like this answered as well.

Reply:
Seems fine to me. We fill that project with documents from the `sourceFileInfos`, and check that its got files in it on line 243, so if this every fails, we would want a PRISM bug to tell us.